Add brand new Dagster project example to dbt quickstart (#21816)
dagster-io/dagster: docs/content/integrations/dbt/quickstart.mdx

---

> ## Wrap your dbt project with Dagster

Reviewer: I know we use the ["wrap" lingo here](https://docs.dagster.io/integrations/dbt/reference#scaffolding-a-dagster-project-from-a-dbt-project), but maybe we should consolidate on "load"

```suggestion
## Load your dbt project in Dagster
```

Author: Done in [6e2bb9f](https://github.com/dagster-io/dagster/pull/21816/commits/6e2bb9f11e758e37394c532832d1c995590403bb)

<TabGroup>

> <TabItem name="Option 1: Create a brand new Dagster project">
> 
> ### Option 1: Create a brand new Dagster project

Reviewer: Just to remove a word

```suggestion
<TabItem name="Option 1: Create a new Dagster project">

### Option 1: Create a new Dagster project
```

Author: Done in [6e2bb9f](https://github.com/dagster-io/dagster/pull/21816/commits/6e2bb9f11e758e37394c532832d1c995590403bb)

You can create a Dagster project that wraps your dbt project by using the `dagster-dbt` command line interface.

> To use the command, you'll need to provide 2 options - `--project-name`, the name of your Dagster project to be created, and `--dbt-project-dir`, the path to your dbt project. In our example, our Dagster project is named `my-dagster-project`and the relative path to our dbt project is `./my-dbt-project`, meaning that we are in the directory where `my-dbt-project` is located.

Reviewer: We can change this so that the user only has to provide one parameter `--dbt-project-dir`.

We can do this by reading the user's `dbt_project.yml` and using the current name as the project name. Maybe event append `_dagster` to that dbt project name,

Author: Currently, if we don't provide the `--project-name` parameter, the user is prompted for the project name.  Should we update this behaviour?

If so, could we do it in another PR? My goal here was to make the command as explicit as possible to make it easier for the user to understand.

Reviewer: Yeah we should do it in another PR.

```shell
dagster-dbt project scaffold --project-name my-dagster-project --dbt-project-dir ./my-dbt-project --use-experimental-dbt-project
```

> This creates a directory called `my-dagster-project/` inside the current directory. The `my-dagster-project/` directory contains a set of files that define a Dagster project.

Reviewer: nit: let's be explicit about what's happening

```suggestion
This command creates a new directory called `my-dagster-project/` inside the current directory. The new `my-dagster-project/` directory will contain a set of files that define a Dagster project to load the dbt project provided in `./my-dbt-project`.
```

Author: Done in [6e2bb9f](https://github.com/dagster-io/dagster/pull/21816/commits/6e2bb9f11e758e37394c532832d1c995590403bb)
